pallets/gear/src/manager.rs

```diff
@@ -200,7 +218,16 @@ where
                 outcome: ExecutionResult::Success,
             }),
             CoreDispatchOutcome::MessageTrap { message_id, trap } => {
-                let reason = trap.map(|v| v.as_bytes().to_vec()).unwrap_or_default();
+                let reason = if let Some(reason) = trap {
```

Comment:
For some reason, don't like `if let` this time, but it's just about beautifying.

I would recommend to pass that info for `log::info!` channel to follow common logging concept of our node (e.g. `InitFailure` processing in the pallet manager).

And, of course, need to add some emoji for the same reason! 😸

Probably, need to consider about passing for `MessageTrap` event also `ProgramId` to concrete which program has been terminated in logs.